src/libraries/Microsoft.Win32.SystemEvents/src/CompatibilitySuppressions.xml

```diff
@@ -7,4 +7,22 @@
     <Left>lib/netstandard2.0/Microsoft.Win32.SystemEvents.dll</Left>
     <Right>lib/net462/Microsoft.Win32.SystemEvents.dll</Right>
   </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0015</DiagnosticId>
+    <Target>E:Microsoft.Win32.SystemEvents.EventsThreadShutdown:[T:System.ObsoleteAttribute]</Target>
+    <Left>ref/net10.0/Microsoft.Win32.SystemEvents.dll</Left>
+    <Right>runtimes/win/lib/net10.0/Microsoft.Win32.SystemEvents.dll</Right>
+  </Suppression>
```

Comment:
What was the error that made you add these lines? I don't think those should be necessary.

Reply:
```
C:\Users\lonitra\.nuget\packages\microsoft.dotnet.apicompat.task\10.0.100-alpha.1.24472.4\build\Microsoft.DotNet.ApiCom
pat.ValidateAssemblies.Common.targets(16,5): error : API compatibility errors between 'ref/net10.0/Microsoft.Win32.Syst
emEvents.dll' (left) and 'runtimes/win/lib/net10.0/Microsoft.Win32.SystemEvents.dll' (right): [C:\Users\lonitra\source\
repos\runtime\src\libraries\Microsoft.Win32.SystemEvents\src\Microsoft.Win32.SystemEvents.csproj]
C:\Users\lonitra\.nuget\packages\microsoft.dotnet.apicompat.task\10.0.100-alpha.1.24472.4\build\Microsoft.DotNet.ApiCom
pat.ValidateAssemblies.Common.targets(16,5): error CP0015: Cannot change arguments of attribute 'System.ObsoleteAttribu
te' on 'Microsoft.Win32.SystemEvents.EventsThreadShutdown'. [C:\Users\lonitra\source\repos\runtime\src\libraries\Micros
oft.Win32.SystemEvents\src\Microsoft.Win32.SystemEvents.csproj]
C:\Users\lonitra\.nuget\packages\microsoft.dotnet.apicompat.task\10.0.100-alpha.1.24472.4\build\Microsoft.DotNet.ApiCom
pat.ValidateAssemblies.Common.targets(16,5): error : API compatibility errors between 'ref/net8.0/Microsoft.Win32.Syste
mEvents.dll' (left) and 'runtimes/win/lib/net8.0/Microsoft.Win32.SystemEvents.dll' (right): [C:\Users\lonitra\source\re
pos\runtime\src\libraries\Microsoft.Win32.SystemEvents\src\Microsoft.Win32.SystemEvents.csproj]
C:\Users\lonitra\.nuget\packages\microsoft.dotnet.apicompat.task\10.0.100-alpha.1.24472.4\build\Microsoft.DotNet.ApiCom
pat.ValidateAssemblies.Common.targets(16,5): error CP0015: Cannot change arguments of attribute 'System.ObsoleteAttribu
te' on 'Microsoft.Win32.SystemEvents.EventsThreadShutdown'. [C:\Users\lonitra\source\repos\runtime\src\libraries\Micros
oft.Win32.SystemEvents\src\Microsoft.Win32.SystemEvents.csproj]
C:\Users\lonitra\.nuget\packages\microsoft.dotnet.apicompat.task\10.0.100-alpha.1.24472.4\build\Microsoft.DotNet.ApiCom
pat.ValidateAssemblies.Common.targets(16,5): error : API compatibility errors between 'ref/net9.0/Microsoft.Win32.Syste
mEvents.dll' (left) and 'runtimes/win/lib/net9.0/Microsoft.Win32.SystemEvents.dll' (right): [C:\Users\lonitra\source\re
pos\runtime\src\libraries\Microsoft.Win32.SystemEvents\src\Microsoft.Win32.SystemEvents.csproj]
C:\Users\lonitra\.nuget\packages\microsoft.dotnet.apicompat.task\10.0.100-alpha.1.24472.4\build\Microsoft.DotNet.ApiCom
pat.ValidateAssemblies.Common.targets(16,5): error CP0015: Cannot change arguments of attribute 'System.ObsoleteAttribu
te' on 'Microsoft.Win32.SystemEvents.EventsThreadShutdown'. [C:\Users\lonitra\source\repos\runtime\src\libraries\Micros
oft.Win32.SystemEvents\src\Microsoft.Win32.SystemEvents.csproj]
C:\Users\lonitra\.nuget\packages\microsoft.dotnet.apicompat.task\10.0.100-alpha.1.24472.4\build\Microsoft.DotNet.ApiCom
pat.ValidateAssemblies.Common.targets(16,5): error : API breaking changes found. If those are intentional, the APICompa
t suppression file can be updated by rebuilding with '/p:ApiCompatGenerateSuppressionFile=true' [C:\Users\lonitra\sourc
e\repos\runtime\src\libraries\Microsoft.Win32.SystemEvents\src\Microsoft.Win32.SystemEvents.csproj]
```

It is essentially the same errors I am getting for current build failure: https://dev.azure.com/dnceng-public/public/_build/results?buildId=852519&view=results